packages/transaction-controller/src/TransactionController.ts

```diff
@@ -521,16 +530,37 @@ export class TransactionController extends BaseControllerV1<
     this.publish =
       hooks?.publish ?? (() => Promise.resolve({ transactionHash: undefined }));
 
+    this.onNetworkStateChange = onNetworkStateChange;
+    this.isMultichainEnabled = isMultichainEnabled;
+    this.incomingTransactions = incomingTransactions;
+    this.getNetworkClientRegistry = getNetworkClientRegistry;
```

Comment:
These should be private, since they are only used internally:
```suggestion
    this.#onNetworkStateChange = onNetworkStateChange;
    this.#isMultichainEnabled = isMultichainEnabled;
    this.#incomingTransactions = incomingTransactions;
    this.#getNetworkClientRegistry = getNetworkClientRegistry;
```